static/src/javascripts/boot.js

```diff
@@ -1,5 +1,12 @@
 // @flow
 
+// es7 polyfills not provided by pollyfill.io
+import 'core-js/modules/es7.object.values';
+import 'core-js/modules/es7.object.entries';
+import 'core-js/modules/es7.object.get-own-property-descriptors';
+import 'core-js/modules/es7.string.pad-start';
+import 'core-js/modules/es7.string.pad-end';
```

Comment:
I updated the PR to ensure padEnd and padStart are not enumerable. Will run the test suite and have it in the next release :-)